backend/src/api/routes/postRoutes.ts

```diff
@@ -1,12 +1,13 @@
 import { Router, Request } from 'express';
 import { run } from '../../common/utils/routeHelper';
-import { addPost, editPost, addComment, getPostComments } from '../../services/postService';
+import { addPost, editPost, addComment, getPostComments, getPostsByUserId } from '../../services/postService';
 
 const router = Router();
 
 router
   .get('/:id/comments', run((req: Request) => getPostComments(req.params.id)))
   .post('/:id/comments', run((req: Request) => addComment(req.user.id, req.params.id, req.body)))
+  .get('/:id/:activeworkspaceid/threads', run((req: Request) => getPostsByUserId(req.params.id, req.params.activeworkspaceid)))
```

Comment:
you don't follow the rules! It should be REST. We do not have threads in business logic

Reply:
activeworkspaceid why is it lowercase?